azerothcore/azerothcore-wotlk
# fix(DB/item_loot): Remove various incorrect items from Mithril Lockbox

## Changes Proposed:
Deletes 25 items of various categories from the [Mithril Lockbox](https://tbc.wowhead.com/item=5758/mithril-lockbox#contains). These are mostly food, herbs, potions, and some miscellaneous quest items that definitely shouldn't be in there. The Lockbox should only contain bags, weapons, gems, armour, and recipes/plans.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/8833
- Closes https://github.com/chromiecraft/chromiecraft/issues/1861

## SOURCE:

https://tbc.wowhead.com/item=5758/mithril-lockbox#contains;150+14+16-19+1
https://web.archive.org/web/20100419042033/http://www.wowhead.com:80/item=5758

## Tests Performed:

- Ran SQL, no errors/warnings, correct number of rows removed.
- Ran query below, no incorrect items found.

## How to Test the Changes:


```SQL
SELECT it.entry, it.name, it.class, it.ItemLevel
FROM `item_loot_template` ilt
JOIN `item_template` it ON it.entry = ilt.item
WHERE ilt.Entry = 5758
AND it.class NOT IN (1, 2, 3, 4, 9)
```
This should return an empty set.

## Known Issues and TODO List:


N/A.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
